compliance/aws/disallowed_regions/aws_disallowed_regions.pt

```diff
@@ -170,53 +161,62 @@ script "js_regions_map", type: "javascript" do
     }
     for (var j = 0; j < param_disallowed_region_lower.length; j++){
       disallowed_region = param_disallowed_region_lower[j];
-        regions_map.push({
-         "region": disallowed_region
-        })
+      regions_map.push({
+        "region": disallowed_region
+      })
     }
   EOS
 end
 
 ###############################################################################
-# Escalations
+# Policy
 ###############################################################################
 
-escalation "esc_report_disallowed_instances" do
-  email $param_email
-end
+policy "pol_list_aws_instances" do
+  validate $ds_list_aws_instances do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} AWS Instances in Disallowed regions"
 
-escalation "esc_terminate_instances_disallowed_region" do
-  request_approval  do
```

Comment:
see my note on Teams, on POL-232 and POL-206 only, we are not changing the escalation blocks yet.  those will be  in late april.